boards/arm/sam34/arduino-due/src/sam_bringup.c

```diff
@@ -116,6 +124,18 @@ int sam_bringup(void)
     }
 #endif
 
-  UNUSED(ret);
+#ifdef HAVE_LEDS
+  board_userled_initialize();
+
+  /* Register the LED driver */
+
+  ret = userled_lower_initialize(LED_DRIVER_PATH);
+  if (ret < 0)
+    {
+      syslog(LOG_ERR, "ERROR: userled_lower_initialize() failed: %d\n", ret);
+      return ret;
+    }
+#endif
```

Comment:
```suggestion
#endif

  UNUSED(ret);
```

Reply:
Please fix this and we can move to merge

Reply:
Fixed! :slightly_smiling_face: